qulice-findbugs/src/main/java/com/qulice/findbugs/FindBugsValidator.java

```diff
@@ -176,14 +176,22 @@ private String excludes(final Environment env,
      * Creates xml with exclude patterns in findbugs native format.
      * @param excludes Exclude patterns
      * @return XML with findbugs excludes
+     * @todo #350 It's better to build XML using Xembly, but when it's
```

Comment:
it's weird, because I tried to add xemble to the pom and use in. Build is green. Could you please provide some logs with the issue? What version of xemble have you tried?